Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -725,6 +753,18 @@ theorem iInf_toSubmodule {ι : Sort*} (S : ι → Subalgebra R A) :
     toSubmodule (⨅ i, S i) = ⨅ i, toSubmodule (S i) :=
   SetLike.coe_injective <| by simp
 
+@[simp]
+theorem iInf_toSubsemiring {ι : Sort*} (S : ι → Subalgebra R A) :
+    (iInf S).toSubsemiring = ⨅ i, (S i).toSubsemiring := by
+  simp only [iInf, sInf_toSubsemiring, ← Set.range_comp]
+  rfl
```

Comment:
```suggestion
  simp only [iInf, sInf_toSubsemiring, ← Set.range_comp, Function.comp]
```

Reply:
Unfortunately, that's not working after merge master.

Reply:
Try with `Function.comp_def` instead, please